src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -853,8 +852,6 @@ protected synchronized void doRun() throws Exception {
                             cleanupOldBundles(bundleDir, file);
                         } catch (Throwable t) {
                             logger.log(Level.WARNING, "Could not save support bundle", t);
-                        } finally {
-                            SecurityContextHolder.setContext(old);
```

Comment:
is this done when `ACLContext` is closed?

Reply:
Yes. Have a look [here](https://github.com/jenkinsci/jenkins/blob/jenkins-2.198/core/src/main/java/hudson/security/ACLContext.java#L64).